module/userProjects/validator/v1.js

```diff
@@ -28,9 +28,8 @@ module.exports = (req) => {
 			req.checkParams('_id').exists().withMessage('required project id')
 		},
 		certificateCallback: function () {
-			req.checkBody('data').exists().withMessage('data is required')
-			req.checkBody('data.transactionId').exists().withMessage('transactionId is required')
-			req.checkBody('data.osid').exists().withMessage('osid is required')
+			req.checkParams('_id').exists().withMessage('required project id')
```

Comment:
@Prajwal17Tunerlabs update this

Reply:
removed validation check for userId and projectId